atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolTest.java

```diff
@@ -41,7 +41,7 @@
 import com.palantir.common.base.FunctionCheckedException;
 
 public class CassandraClientPoolTest {
-    public static final int POOL_REFRESH_INTERVAL_SECONDS = 10;
+    public static final int POOL_REFRESH_INTERVAL_SECONDS = 3 * 60;
```

Comment:
@gsheasby the default is actually `5* 60` (which we can use for consistency if you want). I mainly increased this because the `poolRefresh` was causing issues with connection timeout tests. In the pool creation helper methods, we add servers to the current pool but not the servers in the config, the so auto-reload removes them from under us. Not sure why we did that in the first place but didn't want to mess too much with the other tests.